apps/communications/dialer/test/marionette/PO_test.js

```diff
@@ -0,0 +1,77 @@
+/* global require, marionette, setup, suite, test*/
+'use strict';
+
+var assert = require('chai').assert;
+
+var Dialer = require('./lib/dialer.js');
+//var CallLogView = require('./lib/views/call_log/views.js');
+var DialerOptionsView = require('./lib/views/dialer_options/views.js');
+
+marionette('Dialer > Keypad', function() {
+
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'focusmanager.testmode': true
```

Comment:
Based on [this recent email](https://groups.google.com/d/msg/mozilla.dev.fxos/6gumnQl3AOo/7zhdtiS8AQAJ). You don't need to add this pref anymore.